Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:3.0.2-alpine3.12 AS builder
+FROM ruby:3.0.4-alpine3.15 AS builder
```

Comment:
So according to https://endoflife.date/alpine this looks right r.e build numbers and potentially we could even go so far as to bump it to 3.16 to avoid some further potential hassle, however I believe last time I mentioned bumping Alpine you said there were some dependency issues @jmartin-r7? Do you have more info on this? Happy to do some more digging and investigation given we shouldn't be using a EOL distro for our Docker images but we don't exactly want to start breaking things either.